cli/command/cli_options.go

```diff
@@ -115,3 +115,12 @@ func WithAPIClient(c client.APIClient) CLIOption {
 		return nil
 	}
 }
+
+// WithContext sets the base context for the cli
+// This is used internally for operations may require a context to propagate tracing.
+func WithContext(ctx context.Context) CLIOption {
+	return func(cli *DockerCli) error {
+		cli.baseCtx = ctx
+		return nil
+	}
+}
```

Comment:
This is just a clone of `WithBaseContext` by another name

Reply:
LOLZ You're right! I forgot that was added already; I created this one because I didn't want to extend the DockerCli signature further.

Yup; this can be removed and we can use that one instead 👍